themes/bootstrap5/templates/developersettings/generateapikey.phtml

```diff
@@ -0,0 +1,16 @@
+<?php
+  // Set up breadcrumbs:
+  $this->breadcrumbs()->set($this->translate('Your Account'), $this->url('myresearch-home'))
+    ->add($this->translate('Profile'), active: true);
+?>
+<?= $this->component('page-title', ['title' => $this->translate('Developer::api_key_generate')]); ?>
+
+<?= $this->flashmessages() ?>
+
+<form method="POST" action="<?= $this->url('developersettings-generateapikey') ?>" name="generateAPIKey" title="<?= $this->transEscAttr('Developer::api_key_generate') ?>" data-lightbox-onclose="VuFind.refreshPage">
+  <div class="form-group">
+    <label class="form-label" for="api-key-title"><?=$this->transEsc('Identifier')?>:</label>
```

Comment:
Why label this as "Identifier" rather than "Title"?

(In an ideal world, maybe it would be better to add a whole new translation string for this -- but it's too late to do that right now since translation is already underway).